src/App.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function filtered(title: string, substring: string) {
+  const transformString = title.toLowerCase();
+  const transformSubstring = substring.toLowerCase().trim();
+
+  return transformString.includes(transformSubstring);
+}
+
+function showMovies(query: string) {
```

Comment:
Consider renaming `showMovies` function to a more descriptive name, like `getFilteredMovies` or `filterMovies`. 🤔